docs/LINUX.md

```diff
@@ -91,9 +91,11 @@ The agent uses [**systemd**](https://systemd.io/) to run as a Linux daemon. The
 ### _Configure the agent_
 
 ---
-**NOTE** for users upgrading /migrating from the [legacy LogDNA
+**NOTES**
+* For users upgrading /migrating from the [legacy LogDNA
```

Comment:
nit: remove the space before the slash or just say "or"

Reply:
Thanks for catching that extra space!